### PR TITLE
A/B Test: Concierge Quickstart session upsell to users after plan purchase

### PR DESCRIPTION
**Pull request by victorespigares**
#### Changes proposed in this Pull Request

* Cloned `ConciergeSessionNudge` component to `ConciergeQuickstartSession` for the new AB test with new copy text. The cloned component also includes some (minor) design tweaks (headers, removed overflow:hidden, etc).

#### Testing instructions

Start Calypso locally with this branch and visit http://calypso.localhost:3000/.

1. Put yourself into the `variantQuickstartSession` test group for `conciergeQuickstartSession`.

Screen Shot on 2019-04-10 at 13:02:19

2. Create a new site (to make sure the last redirect is consistent, **don't** choose "_Sell products or collect payments_" for the "_What’s the primary goal..._" question) or use an existing test site.

3. Purchase or upgrade to a plan (no domain).

4. After purchase, confirm you're shown the Concierge upsell page with the `Quick Start` copy.

image

5. Click the button: "Yes, I want a WordPress Expert by my side". You should be taken to the checkout with the session added to the cart.

Screen Shot 2019-04-10 at 13 03 25

6. Purchase the session.

7. Confirm you're either taken to the checklist or to the Thank you page.

screen shot 2019-01-02 at 11 21 46 am

Screen Shot 2019-04-10 at 13 38 59

-------------

Next, put yourself into the `controlSupportSession` test group and run through steps 2-4 again.
This time you should be shown the Concierge upsell page with the control copy:

Screen Shot 2019-04-10 at 13 02 57

#### Comments

I'm just now realizing I should've committed the cloned component earlier, so real changes would have shown up in the diff, instead of the whole file. Apologies!

Here are the diffs between `ConciergeSessionNudge` files (left) and `ConciergeQuickstartSession` (right): 

* [index.jsx](https://www.diffchecker.com/nSynBbd8)
* [style.scss](https://www.diffchecker.com/J1SgLHb6)

#### Questions

* ~~How were translations/audiences dealt with in the previous test? Are we running this AB test only to English audiences?~~ Just answered this by actually paying attention to the [AB test README](https://github.com/Automattic/wp-calypso/tree/master/client/lib/abtest). Short answer: By default, tests only run on users where the locale is set to English.
* This test is supposed to exclude the GSuite version of the flow. I've applied the `abtest()` logic [here](https://github.com/Automattic/wp-calypso/pull/32176/files#diff-199e3967ae413c2ff48db966abbf8386) for the non-eligible for GSuite flow, which should be ok, but would appreciate another pair of eyes here.
* Also related: can't help but noticed that [there's an abtest() call](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/checkout/checkout/index.jsx#L420) which looks like leftovers from the previous Concierge test for GSuite. Is this ok to remove?
* Noob question: after the review process, is deploying these changes enough to have the AB test already running? or is there anything else missing?

### CODE REVIEW

**Comment by michelleweber** [comment]
Suggested edit:

**Create with confidence -- your personal WordPress expert will help**

Get the vision that's in your head onto the screen, fast --  with an expert who's helped hundreds of people built their dream sites. 

**Introducing WordPress.com Quick Start sessions.**

Quick Starts are 45-minute, one-on-one conversations with a WordPress.com support expert. We call them Happiness Engineers: they know WordPress.com inside and out, and their job is to make sure your website puts a smile on your face.

During your Quick Start, a Happiness Engineer will give you a personalized tour of WordPress.com and offer pro advice on everything from:
- Choosing the perfect design
- Customizing your site with your own branding, images, colors, and fonts
- Adding pages and posts to your site -- and what pages are most important
- Finding your audience, and getting traffic, followers, and likes on your new site

Focus on a particular part of the WordPress dashboard, get an overview of WordPress.com, drill down on design. What you cover in a Quick Start is up to you! And of course, you can ask any and all questions you have about your new site. 

At the end, you'll have a solid foundation for continuing to build the site you've always wanted -- and the confidence to do it.

Ready? Book your Quick Start session today for _$98.00_ $49.00.

(This is a one-time offer; we want to help you make the most of your new plan! After today, Quick Start sessions are $98.00.)

_Note: Quick Start sessions are currently available only in English._

File: client/my-sites/checkout/concierge-quickstart-session/index.jsx [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+							{ translate(
+								"{{i}}Quick Start{{/i}} sessions are 45-minute one-on-one conversations between you and one of our website building experts. They know WordPress inside out and will help you achieve your goals with a smile. That's why we call them Happiness Engineers.",
+								{
+									components: { i: <em /> },
+								}
+							) }
+						</p>
+						<p>
+							{ translate(
+								'On these sessions, a Happiness Engineer will show you everything you need to build a great site in the fastest way possible, including:'
```
Comment (travisw) [inline client/my-sites/checkout/concierge-quickstart-session/index.jsx]: This copy kind of promises the session will go over all of these things. Even though they're really more customized to what the user wants.
Do you think the HEs doing these session will need to know that the user saw this upsell page vs the control?

Comment (travisw) [reply client/my-sites/checkout/concierge-quickstart-session/index.jsx]: Also, feel free to ignore any comments I leave on the copy. Just my 2¢.

File: client/my-sites/checkout/concierge-quickstart-session/index.jsx [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+	}
+
+	body() {
+		const { translate, productCost, productDisplayCost, currencyCode } = this.props;
+		const fullCost = Math.round( productCost * 2 );
+		//const savings = fullCost - productCost;
+		return (
+			<Fragment>
+				<h4 className="concierge-quickstart-session__sub-header">
+					{ translate( 'Presenting… a personal WordPress Expert, by your side' ) }
```
[the remark this answers is not in the document]
Comment (victorespigares) [reply client/my-sites/checkout/concierge-quickstart-session/index.jsx]: Yeah, good point. Tried both, and IMO it kind of reads better without the ".com", but as you said, it sort of works either way.

File: client/my-sites/checkout/concierge-quickstart-session/index.jsx (hunk 1 above)
Comment (victorespigares) [reply client/my-sites/checkout/concierge-quickstart-session/index.jsx]: Also good points. 

The assumption here is that is too early for people to see the value of the session (i.e. they haven't faced any real issues yet) that's why the copy kind of leans into painting a more "guided" picture. I've touched a little bit the copy to balance this. 

Also, yes, HEs will get up to speed to know this test is running.

**Comment by victorespigares** [comment]
> Nice job on this @victorespigares - it's looking pretty good. I've left a few comments.

Awesome, thanks! Having the previous test you did as a template was a great starting point, so actually, I was standing on the shoulders of giants here! Good point with the leftover comments and thanks as well for the answers.

> Since this adds a new ab test that affects certain flows, we'll also need to create a small PR and merge it in the [wp-e2e-tests](https://github.com/Automattic/wp-e2e-tests) repository. Here's an example: [Automattic/wp-e2e-tests#1799](https://github.com/Automattic/wp-e2e-tests/pull/1799)

Great, thanks for the example, very helpful. I've already submitted the PR here: https://github.com/Automattic/wp-e2e-tests/pull/1829

> If we end up testing a lot of different copy for pages like this in the future, we should figure out a way to minimize all the duplicated code. But I think it's fine this way for now.

My same thought :D

> You're right, that's some leftover code that isn't currently being used. It can be removed, along with the [test definition](https://github.com/Automattic/wp-calypso/blob/master/client/lib/abtest/active-tests.js#L75). But that cleanup work should be done in a separate PR to keep this one more focused.

Great, will be doing that in a separate PR.

**Comment by victorespigares** [comment]
@michelleweber: Thanks a lot for the suggested copy! 

I wasn't really aware what the "Needs copy review" label will "triggered," but I've ended up using some of your text for the current version and it's always thoughtful to see another take on this. 

We might end up using more of the angle in your text in future tests, so thanks a lot!

File: client/my-sites/checkout/concierge-quickstart-session/index.jsx [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+									components: { i: <i /> },
+								}
+							) }
+						</p>
+						<p>
+							<b>
+								{ translate(
+									'Book your {{i}}Quick Start{{/i}} session below at a special one-time price of {{del}}%(oldPrice)s{{/del}} %(price)s.',
+									{
+										components: { del: <del />, i: <i /> },
```
[the remark this answers is not in the document]
Comment (victorespigares) [reply client/my-sites/checkout/concierge-quickstart-session/index.jsx]: Switched to `em` 👍

**Pull request merged**